src/senaite/core/exportimport/instruments/biodrop/ulite/__init__.py

```diff
@@ -46,7 +46,7 @@ def _parseline(self, line):
             this_row = line.split(",")
             # if less values are found than headers, it's an error
             if len(this_row) != len(self.data_header):
-                self.err("Data row number " + item_nr + " has the wrong number of items")
+                self.err("Data row number " + self._numline + " has the wrong number of items")
```

Comment:
`item_nr` does not exist anywhere in the code base but `_numline` contains the row index which seems to be desired here